Untitled Document

```diff
@@ -0,0 +1,2 @@
+REACT_APP_SERVER_ADDRESS=http://humci.pih-emr.org/
```

Comment:
We don't want this file to be committed to the source directory, as may be different for each user.